src/main/java/io/supertokens/inmemorydb/queries/TOTPQueries.java

```diff
@@ -0,0 +1,181 @@
+package io.supertokens.inmemorydb.queries;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import io.supertokens.inmemorydb.Start;
+import io.supertokens.inmemorydb.config.Config;
+import io.supertokens.pluginInterface.RowMapper;
+import io.supertokens.pluginInterface.exceptions.StorageQueryException;
+import io.supertokens.pluginInterface.exceptions.StorageTransactionLogicException;
+
+import io.supertokens.pluginInterface.totp.TOTPDevice;
+import io.supertokens.pluginInterface.totp.TOTPUsedCode;
+
+import static io.supertokens.inmemorydb.QueryExecutorTemplate.execute;
+import static io.supertokens.inmemorydb.QueryExecutorTemplate.update;
+
+public class TOTPQueries {
+    public static String getQueryToCreateUserDevicesTable(Start start) {
+        return "CREATE TABLE IF NOT EXISTS" + Config.getConfig(start).getTotpUserDevicesTable() + " ("
+                + "user_id VARCHAR(128) NOT NULL," + "device_name VARCHAR(256) NOT NULL,"
+                + "secret_key VARCHAR(256) NOT NULL,"
+                + "period INTEGER NOT NULL," + "skew INTEGER NOT NULL," + "verified BOOLEAN NOT NULL,"
```

Comment:
Do we use BOOLEAN in other tables as well?

Reply:
Nope. Searched for `BOOLEAN` found only TOTP recipe.